playbooks/robusta_playbooks/pod_troubleshooting.py

```diff
@@ -256,6 +256,110 @@ def get_debugger_warnings(data):
         return None
     return message
 
+@action
+def debugger_stack_trace(event: PodEvent, params: DebuggerParams):
+    """
+    Prints a stack track of a python process and child threads
+
+    Create a finding with the stack trace results.
+    """
+    pod = event.get_pod()
+    if not pod:
+        logging.info(f"stack_trace - pod not found for event: {event}")
+        return
+
+    if not params.pid:
```

Comment:
Is pid really mandatory?
Can you ask just as pod stack trace, without a pid filter? (some pods may have only 1 process )